submitit/conftest.py

```diff
@@ -8,3 +8,8 @@
 @pytest.fixture()
 def executor(tmp_path: Path) -> LocalExecutor:
     return LocalExecutor(tmp_path)
+
+
+@pytest.fixture(params=["a_0", "a 0", 'a"=0"', "a'; echo foo", r"a\=0", r"a\=", "a\n0"])
+def weird_dir(request) -> str:
+    return request.param
```

Comment:
you could send a temporary path instead directly :D

Reply:
good idea ! done